src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,212 @@
+    - To set up each cluster, you can follow [Deploy CockroachDB on Premises]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}).
+    - The [Deploy CockroachDB on Premises]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}) tutorial creates a self-signed certificate for each {{ site.data.products.core }} cluster. To create certificates signed by an external certificate authority, refer to [Create Security Certificates using OpenSSL]({% link {{ page.version.version }}/create-security-certificates-openssl.md %}).
+    - Both clusters can be empty.
+    - One cluster can be empty and the other can have an existing table/dataset.
+
+    {{site.data.alerts.callout_info}}
+    If you need to run LDR through a load balancer, use the load balancer IP address as the SQL advertise address on each cluster. It is important to note that using a load balancer with LDR can impair performance.
+    {{site.data.alerts.end}}
+
+- All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Connect from the destination to the source](#step-2-connect-from-the-destination-to-the-source).
+- If both clusters are empty, create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters. If one cluster already has an existing table that you'll replicate, ensure the other cluster's table definition matches. For more details on the supported schemas, refer to [Schema Validation](#schema-validation).
+
+To create bidirectional LDR, you can complete the [optional step](#step-4-optional-set-up-bidirectional-ldr) to start the second LDR job that sends writes from the table on cluster B to the table on cluster A.
+
+### Schema validation
+
+Before you start LDR, you must ensure that all column names, types, constraints, and unique indexes on the destination table match with the source table.
+
+You cannot use LDR on a table with a schema that contains the following:
+
+- Columns with [user-defined types]({% link {{ page.version.version }}/create-type.md %})
+- [Column families]({% link {{ page.version.version }}/column-families.md %})
+- [Partial indexes]({% link {{ page.version.version }}/partial-indexes.md %})
+- Indexes with a [computed column]({% link {{ page.version.version }}/computed-columns.md %})
+- Composite types in the [primary key]({% link {{ page.version.version }}/primary-key.md %})
+
+For more details, refer to the LDR [Known limitations]({% link {{ page.version.version }}/set-up-logical-data-replication.md %}#known-limitations).
+
+When you run LDR in `immediate` mode, you cannot replicate a table with [SQL constraints]({% link {{ page.version.version }}/constraints.md %}). In `validated` mode, SQL constraints **must** match. 
```

Comment:
SQL constraints --> it should be "you cannot replicate a table with foreign key constraints". Other constraints are fine.